.probo.yaml

```diff
@@ -45,6 +46,7 @@ steps:
   - name: Post-Installation Steps
     plugin: Script
     script:
+      - $SRC_DIR/az-quickstart-scaffolding/vendor/bin/drush -y -n pm:install upgrade_status
```

Comment:
```suggestion
```

Reply:
I think we still want the pm:install step.

Reply:
I have another PR for this, so we can separate this point of contention (adds 14 minutes to builds) out https://github.com/az-digital/az_quickstart/pull/2111

Reply:
However, that PR is not against this branch, so either way is fine with me